sci-electronics/systemc/systemc-2.3.0-r1.ebuild

```diff
@@ -45,7 +45,8 @@ src_prepare() {
 }
 
 src_configure() {
-	econf $(use_enable static-libs static) CXX=$(tc-getCXX)
```

Comment:
I removed them because they use ${CHOST}-g++ regardless of the existence of `CXX=$(tc-getCXX)`, at least at 2.3.0 and 2.3.1. However, indeed, there must be any reason why the argument was added. I will restore them for now and check again later.